apm-agent-core/src/main/java/co/elastic/apm/agent/impl/error/ErrorCapture.java

```diff
@@ -146,6 +146,32 @@ public TraceContext getTraceContext() {
         return traceContext;
     }
 
+    @Override
+    public Span createSpan() {
+        throw new UnsupportedOperationException();
```

Comment:
```suggestion
        throw new UnsupportedOperationException("Creating a span as a child of an error is not possible");
```

Reply:
added message

Reply:
Also, add similar messages to the other methods

Reply:
added